### PR TITLE
[MonologBridge] RouteProcessor class is now final to ease the the removal of deprecated event

### PR DESCRIPTION
**Pull request by Simperfit**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | none  
| License       | MIT
| Doc PR        | none  

To ease the removal of deprecated events this class has been made final as said in https://github.com/symfony/symfony/pull/31672#discussion_r288479232

### CODE REVIEW

File: UPGRADE-4.4.md [hunk 1]
```diff
@@ -19,3 +19,8 @@ DependencyInjection
      my_service:
        factory: ['@factory_service', method]
    ```
+
+Monolog Bridge
+--------------
+
+* The `RouteProcessor` marked final.
```
Comment (nicolas-grekas) [inline UPGRADE-4.4.md]: has been

Comment (nicolas-grekas) [reply UPGRADE-4.4.md]: don't miss updating the 5.0 file

**Comment by Simperfit** [comment]
done @nicolas-grekas

**Comment by nicolas-grekas** [comment]
Thank you @Simperfit.

**Pull request merged**